app/mesh/changelog.md

```diff
@@ -4,6 +4,52 @@ no_search: true
 no_version: true
 ---
 
+## 1.8.0
+
+> Released on **TBD**
+
+Built on top of [Kuma 1.7.0](https://github.com/kumahq/kuma/releases/tag/1.7.0)
+
+### Changes
+
+New Features:
+
+- Support for arm64
+- Graceful shutdown of OPA
+- Role-based AWS auth for Vault
+- Add Vault AWS Auth option to set Server ID
```

Comment:
```suggestion
- Role-based AWS authentication for Vault
- Added a Vault AWS authentication option to set the server ID
```
What is the option? Can you add it as: "Added the `<option-name>` Vault AWS authentication option to set the server ID."

Reply:
I believe the option is called `Server ID` :-)

Reply:
> Added a Vault AWS authentication option to set the server ID header

Do you think it's ok now?